lib/modules/xPostLinks.js

```diff
@@ -0,0 +1,66 @@
+addModule('xPostLinks', function(module, moduleID) {
+	module.moduleName = 'X-post Links';
+	module.category = 'Posts';
+	module.description = 'Create links to x-posted subreddits in the taglines.';
+	module.include = [
+		/^https?:\/\/([a-z]+)\.reddit\.com/i
+	];
+
+	module.go = function() {
+		if ((module.isEnabled()) && (module.isMatchURL())) {
+
+			module.createLinks();
+			window.addEventListener('neverEndingLoad', module.createLinks);
+		}
+	};
+
+	module.createLinks = function () {
+		var
+		titles        = document.querySelectorAll('a.title'),
```

Comment:
Could you implement the `= function(container) { var titles = (container || document.body).querySelectorAll(...)` pattern? helps avoid re-scanning elements that have already been processed.